### DISCUSSION TITLE
Color blend "hidden" feature

### DISCUSSION THREAD

**Discussion by buzzmandt** [inkstitch/inkstitch#1511] [Show and tell]
Have been trying out the gradient/color blend option. Results seem pretty good.
![IMG_20220101_082446419](https://user-images.githubusercontent.com/76489837/147854831-3f66dd0a-71c8-492a-835a-23c266d46631.jpg)

**Comment by lexelby** [comment]
Wow, that looks great!  Thanks for sharing.
